src/.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
       - id: check-yaml
 
   - repo: https://github.com/psf/black
-    rev: 21.8b0
+    rev: 22.3.0
```

Comment:
I have noticed [they have a tag named "stable"](https://github.com/psf/black/tags) that might always point to the latest version in case that helps not needing to update the version here when it changes. The problem is it might introduce problems with a new version change and we won't notice so easily the problem might come from here.

Reply:
Yep, changed it to stable and it still works so I'll go with that. Thanks!